api/src/main/java/marquez/db/JobDao.java

```diff
@@ -60,11 +60,17 @@
   void updateVersion(UUID rowUuid, Instant updatedAt, UUID currentVersionUuid);
 
   String JOB_SELECT =
-      "SELECT j.*, jc.context "
+      "SELECT j.*, jc.context, "
+          + "(SELECT event->'job'->'facets' "
+          + "   FROM lineage_events AS le "
+          + "  WHERE jv.latest_run_uuid::text = le.run_id "
+          + "  ORDER BY event_time DESC "
+          + "  LIMIT 1) AS facets "
```

Comment:
I think the lineage events are supposed to be cumulative for a run. So it's conceivable that a user may post multiple partial lineage events, with each event contributing to the whole. This `LIMIT 1` assumes the latest event contains all of the information, which isn't necessarily so, right?